site2/website/releases.json

```diff
@@ -1,5 +1,6 @@
 [
   "2.10.1",
+  "2.7.5",
```

Comment:
1. Should this change be made to line 13 (below 2.8.0) instead of here?
2. Should `release.json` and `version.json` be updated after 2.7.5 is released?

Reply:
Correct! The `site2/website/versions.json` file needs an update as well. @Jason918

Reply:
> * Should this change be made to line 13 (below 2.8.0) instead of here?

There is a note in release process doc:

> Note: You can move the latest version under the old version in the versions.json file. Make sure the Algolia index works before moving 2.X.0 as the current stable.

I am not quite sure I understand it correctly. Just take reference from https://github.com/apache/pulsar/pull/16341/files

Reply:
> Correct! The `site2/website/versions.json` file needs an update as well. @Jason918

Addressed. PTAL. @momo-jun